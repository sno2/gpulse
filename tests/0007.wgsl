const a = 0.e+4f;
const b = 01.;
const c = .01;
const d = 12.34;
const f = .0f;
const g = 0h;
const h = 1e-3;
