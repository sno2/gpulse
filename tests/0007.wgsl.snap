ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 6, .end = 7 }, .typ = null, .value = ast.Node{ .number_literal = ast.Span{ .start = 10, .end = 16 } } } } ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 24, .end = 25 }, .typ = null, .value = ast.Node{ .number_literal = ast.Span{ .start = 28, .end = 31 } } } } ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 39, .end = 40 }, .typ = null, .value = ast.Node{ .number_literal = ast.Span{ .start = 43, .end = 46 } } } } ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 54, .end = 55 }, .typ = null, .value = ast.Node{ .number_literal = ast.Span{ .start = 58, .end = 63 } } } } ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 71, .end = 72 }, .typ = null, .value = ast.Node{ .number_literal = ast.Span{ .start = 75, .end = 78 } } } } ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 86, .end = 87 }, .typ = null, .value = ast.Node{ .number_literal = ast.Span{ .start = 90, .end = 92 } } } } ast.Node{ .const_decl = ast.ConstDecl{ .name = ast.Span{ .start = 100, .end = 101 }, .typ = null, .value = ast.Node{ .number_literal = ast.Span{ .start = 104, .end = 108 } } } } 